cmd/cstor-volume-mgmt/volume/volume.go

```diff
@@ -196,6 +198,57 @@ func CreateIstgtConf(cStorVolume *apis.CStorVolume) ([]byte, error) {
 	return dataBytes, nil
 }
 
+// ResizeTargetVolume sends resize volume command to istgt and get the response
+func ResizeTargetVolume(cStorVolume *apis.CStorVolume) error {
+	index, _, err := FileOperatorVar.GetLineDetails(util.IstgtConfPath, "LUN0 Storage")
+	if err != nil {
+		return errors.Wrapf(err,
+			"failed to get storage details from %s file",
+			util.IstgtConfPath)
+	} else if index == -1 {
+		return errors.Errorf("storage details not found in file %s", util.IstgtConfPath)
+	}
+	// send resize command to istgt and read the response
+	resizeCmd := getResizeCommand(cStorVolume)
+	sockResp, err := UnixSockVar.SendCommand(resizeCmd)
```

Comment:
should we first update the conf file and then send the istgtcontrol resize command?

Reply:
So is there any coroner cases that we left when we do command execution first and then update conf file?

Reply:
So in case of pod restarts anyways istgt.conf file will be updated to latest size(capacity will be picked from cstorvolume.spec which has updated capacity) so no problem in those cases.